inc/saifdb.h

```diff
@@ -114,7 +114,7 @@ typedef enum _sai_fdb_entry_attr_t
      * @flags CREATE_AND_SET
      * @default SAI_PACKET_ACTION_FORWARD
      */
-    SAI_FDB_ENTRY_ATTR_PACKET_ACTION,
+    SAI_FDB_ENTRY_ATTR_PACKET_ACTION = SAI_FDB_ENTRY_ATTR_START + 0x1,
```

Comment:
maybe not use SAI_FDB_ENTRY_ATTR_START , just direct value 0x1, and on other

Reply:
@kcudnik I could fix it and repost the PR, but please see my question above on the errors which would persist even without that: https://github.com/opencomputeproject/SAI/pull/1398#issuecomment-1014988193